src/Common/GeneratorConfig.php

```diff
@@ -205,7 +205,7 @@ public function loadPaths()
         $this->pathViews = config(
             'infyom.laravel_generator.path.views',
             base_path('resources/views/')
-        ).$viewPrefix.$this->mCamelPlural.'/';
+        ).$this->mSnakePlural.'/';
```

Comment:
@tiago-wd why this change?

Reply:
It's changing the view name generation from camel case to snake case